Conversions/DateDayDifference.js

```diff
@@ -19,7 +19,7 @@ const DateToDay = (dd, mm, yyyy) => {
 
 const DateDayDifference = (date1, date2) => {
   // firstly, check that both input are string or not.
-  if (typeof date1 !== 'string' && typeof date2 !== 'string') {
+  if (typeof date1 !== 'string' || typeof date2 !== 'string') {
```

Comment:
this should be "or"

Reply:
indeed; that's technically a bug fix rather than just refactoring, but it's fine